src/libs/banners/banners.ts

```diff
@@ -32,13 +32,24 @@ const getBridgeActionText = (routeStatus: ActiveRoute['routeStatus'], isBridgeTx
   return routeStatus === 'completed' ? 'Swapped' : 'Swap'
 }
 
-const getBridgeBannerText = (route: ActiveRoute, isBridgeTxn: boolean) => {
+const getBridgeBannerText = (route: ActiveRoute, isBridgeTxn: boolean, networks?: Network[]) => {
   const steps = getQuoteRouteSteps(route.route.userTxs)
   const actionText = getBridgeActionText(route.routeStatus, isBridgeTxn)
   const fromAssetSymbol = steps[0].fromAsset.symbol
   const toAssetSymbol = steps[steps.length - 1].toAsset.symbol
 
-  const assetsText = `${fromAssetSymbol} to ${toAssetSymbol}`
+  let assetsText = `${fromAssetSymbol} to ${toAssetSymbol}`
+
+  if (networks) {
+    const fromAssetNetwork = networks.find((n) => Number(n.chainId) === steps[0].fromAsset.chainId)
+    const toAssetNetwork = networks.find(
+      (n) => Number(n.chainId) === steps[steps.length - 1].toAsset.chainId
+    )
+    if (fromAssetNetwork && toAssetNetwork) {
+      assetsText = `${fromAssetSymbol} on (${fromAssetNetwork.name}) to ${toAssetSymbol} on (${toAssetNetwork.name})`
+    }
```

Comment:
Maybe better to either: 1) move "on" inside the network name in parentheses? or 2) Remove the parentheses.

> Bridged USDC on (Optimism) to CBBTC on (Base) (step 2 of 2)